packages/wdio-browserstack-service/src/@types/bstack-service-types.d.ts

```diff
@@ -1,13 +1,14 @@
 declare namespace WebdriverIO {
+    import PercyCaptureMap from '../Percy/PercyCaptureMap.js'
```

Comment:
Why it is not at top?

Reply:
https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html

namespaces/modules are used to organise code and declarations, import statement outside doesn't make sense and hence the build will fail too if we are to import outside

Reply:
Fixed